source/infrastructure/package.json

```diff
@@ -23,6 +23,7 @@
   },
   "dependencies": {
     "@aws-cdk/aws-servicecatalogappregistry-alpha": "^2.62.2-alpha.0",
+    "@aws-cdk/aws-lambda-python-alpha" : "~2.68.0-alpha.0",
     "@aws-solutions-constructs/aws-lambda-dynamodb": "2.30.0",
```

Comment:
match the versions of all the cdk deps

Reply:
lambda-dynamodb construct appears to follow a different version pattern, but I have updated aws-cdk-lib and @aws-cdk/aws-servicecatalogappregistry-alpha accordingly

Reply:
nevermind, updating these is non-trivial (particularly appregistry). I have created a separate story to address this